asm/inline.rs

```diff
@@ -20,45 +22,58 @@ pub unsafe fn __control_r() -> u32 {
 
 #[inline(always)]
 pub unsafe fn __control_w(w: u32) {
-    asm!("msr CONTROL, {}", in(reg) w);
+    // ISB is required after writing to CONTROL,
+    // per ARM architectural requirements (see Application Note 321).
+    asm!("msr CONTROL, {}", "isb", in(reg) w);
```

Comment:
Having each instruction in its own line is much more readable IMO

Reply:
Makes sense, I've reformatted as such.